### PR TITLE
complete rewrite on top of the Snowdrop license generator

### PR DESCRIPTION
**Pull request by Ladicek**

### CODE REVIEW

**Comment by Ladicek** [comment]
Draft because it depends on a SNAPSHOT of the Snowdrop license generator. In other words, waiting for this PR to be merged and released: https://github.com/snowdrop/licenses-generator/pull/13

File: README.md [hunk 1]
```diff
@@ -1,60 +1,35 @@
...
+URL of an external license service that can provide license information.
+If not set, no external license service is used.
 
 ```bash
--Dbooster.product.build=true
+-Dexternal.license.service=http://.../find-license-check-record-and-license-info
 ```
 
-## booster.repo.url
+## product variant of the example?
```
Comment (sberyozkin) [inline README.md]: Is the `?` mark intentional ?

**Comment by sberyozkin** [comment]
It all looks very neat, nice to have a fresher implementation. Will `generate-licenses` script have to be updated ?

**Comment by sberyozkin** [comment]
LGTM, please merge once the question mark question above is resolved

File: README.md (hunk 1 above)
Comment (Ladicek) [reply README.md]: It's meant to be a question, which is later answered, but I agree it can be confusing. I'll rephrase.

**Comment by Ladicek** [comment]
Yea, the `generate_licenses.sh` script needs to be changed. Actually most of it will be removed :-)

**Pull request merged**